Code/GraphMol/ChemReactions/ReactionParser.h

```diff
@@ -79,7 +79,7 @@ ChemicalReaction *RxnDataStreamToChemicalReaction(std::istream &rxnStream,
 ChemicalReaction *RxnSmartsToChemicalReaction(
     const std::string &text,
     std::map<std::string, std::string> *replacements = 0,
-    bool useSmiles = false);
+    bool useSmiles = false, bool sanitize=false);
```

Comment:
This is missing a docstring for sanitize as well. I'd also be very explicit about what exactly sanitization here entails, as it seems to differ depending on context.